packages/db-ui-elements-stencil/src/components/db-textarea/stories/db-textarea.intro.stories.mdx

```diff
@@ -63,4 +63,15 @@ Make textareas look inactive by adding the disabled boolean attribute to any &lt
   ></db-textarea>
 </Canvas>
 
+<db-headline variant="2">visually hide the label</db-headline>
+
+> A label for a form control helps everyone better understand its purpose. In some cases, the purpose may be clear enough from the context when the content is rendered visually. The label can be hidden visually, though it still needs to be provided within the code to support other forms of presentation and interaction, such as for screen reader and speech input users.
+source: <https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text>
+
+You could set the attribute `label-hidden="true"` for the rare case that you would want to visually hide the textual label for a textarea element.
+
+<Canvas>
+<db-textarea label="Semitransparent" name="textarea-name" label-hidden="true"></db-textarea>
```

Comment:
@mfranzke for textarea I can't see any change when disabling the label

Reply:
I've [opened a bug ticket for this aspect](https://github.com/db-ui/elements/issues/2265), could you please approve this ticket still so that we could get it out finally? The request initially was about `input`-elements, and that for we could still investigate on the `textarea` afterwards.